daemon/src/projection.rs

```diff
@@ -389,9 +390,17 @@ impl Cfd {
                 (CfdState::PendingCommit, vec![])
             }
             OracleAttestedPostCetTimelock { cet, price } => {
-                self.details
-                    .tx_url_list
-                    .push(TxUrl::new(cet.txid(), network, TxLabel::Cet));
+                let tx_url = if let Some(dlc) = self.latest_dlc.as_ref() {
+                    TxUrl::from_transaction(
+                        &cet,
+                        &dlc.script_pubkey_for(role),
```

Comment:
Same here. What script is this returning? Do our CETs actually spend to this script?

Reply:
I think we use the same address for all final transactions, so this should be okay if that's the case.